smithy-diff/src/main/java/software/amazon/smithy/diff/evaluators/ChangedEnumTrait.java

```diff
@@ -67,11 +70,20 @@
             }
         }
 
+        int newPosition = 0;
         for (EnumDefinition definition : newTrait.getValues()) {
             if (!oldTrait.getEnumDefinitionValues().contains(definition.getValue())) {
-                events.add(note(change.getNewShape(), String.format(
-                        "Enum value `%s` was added", definition.getValue())));
+                if (newPosition <= oldEndPosition) {
+                    events.add(error(change.getNewShape(), String.format(
+                            "Enum value `%s` was inserted. This can cause compatibility issues when ordinal values are "
```

Comment:
I love the extra bit of context here about why this matters! Can you say something like "Enum value `%s` was inserted, but not at the end of the list of existing enum values. [...]" or something to that effect?